src/richie/apps/courses/models/course.py

```diff
@@ -399,6 +399,18 @@ def state(self):
             self.start, self.end, self.enrollment_start, self.enrollment_end
         )
 
+    def get_course(self):
+        """Get the course for this course run."""
+        node = self.extended_object.node
+        ancestors_paths = [
```

Comment:
This assignment might deserve a comment unless it's regular old DjangoCMS stuff

Reply:
It's inspired from treebeard's `get_ancestors` method: https://github.com/django-treebeard/django-treebeard/blob/master/treebeard/mp_tree.py#L1052.

Maybe I can directly use it... let me try.

Reply:
Done. Much clearer.